common/static/common/js/components/views/list.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Generic view to render a collection.
+ */
+;(function (define) {
+    'use strict';
+    define(['backbone', 'underscore'], function(Backbone, _) {
+        var ListView = Backbone.View.extend({
+            /**
+             * Override with the view used to render models in the collection.
+             */
+            itemViewClass: Backbone.View,
+
+            initialize: function (options) {
+                this.itemViewClass = options.itemViewClass || this.itemViewClass;
+                this.collection.on('add', this.render, this);
```

Comment:
At some point we'll want ``add`` and ``remove`` to not re-render the entire collection. Maybe just add a TODO to this effect.